torchaudio/transforms.py

```diff
@@ -653,12 +657,16 @@ def __init__(self,
                  lowpass_filter_width: int = 6,
                  rolloff: float = 0.99) -> None:
         super(Resample, self).__init__()
```

Comment:
You might want to reuse the logic to issue the warning about non-integer sampling rate values somehow . So that it will be triggered from `T.Resample` as well.

Reply:
also, the `__init__` input types currently indicates an int for the frequencies, but the docstring mention float. which one should be changed?

Reply:
Since `F.resample`. Expects `float`, let's align it with `float`.